src/pages/index.js

```diff
@@ -106,44 +66,108 @@ const settings = {
   errorClass: "modal__error_visible",
 };
 
+/// API classes ///
+
+const apiInstance = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: "ca29b9d7-c085-4996-bfd4-aa6f252bbde8",
+  contentType: "application/json",
+});
+
+apiInstance
+  .getInitialCards() // metodo de api class para obtener las cards
+  .then((cards) => {
+    // puse cards, pero tambien se usa "res"de respuesta
+    console.log(cards);
+    cardSection.renderItems(cards);
+  })
+  .catch((err) => console.error("I got an error:", err.message));
+
+apiInstance // llamo al metodo de la api class para actualizar el usuario
+  .getUserInfo()
```

Comment:
Better to use `Promise.all` to load initial data

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
thank you for the advice, makes my code be more organize